### ISSUE TITLE
grunt v1.6.0 throwing error "SyntaxError: Unexpected token '.'" through npm rimraf package.

### ISSUE THREAD

**Issue by artbindu** [gruntjs/grunt#1752]
# grunt v1.6.0 throwing error "SyntaxError: Unexpected token '.'" through npm rimraf package.

## Some braking changes happening in latest `grunt` changes.

I am using node & npm version as:
```
node: v12.16.1
npm: 6.13.4
```

I am getting errors like:
<img width="890" alt="Screenshot 2023-01-30 at 3 17 48 PM" src="https://user-images.githubusercontent.com/44528087/215501687-4ea3f958-31ab-4ad1-a2d1-84bc2b9551b0.png">

When I see the [`npm rimraf`](https://www.npmjs.com/package/rimraf) they did some changes few days ago.
But at the end of the day, I see the issue is happening from grant version.

---

I see issue is not happening until [grunt v1.5.3](https://www.npmjs.com/package/grunt/v/1.5.3).
Issue is reproducing in latest version: **[grunt v1.6.0](https://www.npmjs.com/package/grunt/v/1.6.0)**

I think the issue may be happening from es5 or es6 syntax related.

**Comment by RobinBol** [comment]
Apparently as of grunt@1.6.x Node 16 or higher is required. Been debugging this for a while myself.. Maybe a major bump would have been better?

**Comment by vladikoff** [comment]
@artbindu could you try your project with node.js 14 or 16? 12 is really outdated.

**Comment by artbindu** [comment]
The issue is still reproducing with node v14. But in node v16 I did not see the issue with the latest grunt version(v1.6.x). 
Temporarily I fixed that with grunt `1.5.3`. 

Thanks @vladikoff, thanks @vladikoff for your suggestion.  

You may close the ticket.

**Comment by vladikoff** [comment]
Fixed in 1.6.1 . thanks for the report!

**Comment by artbindu** [comment]
Yes, it's resolved in [grunt 1.6.1](https://github.com/gruntjs/grunt/releases/tag/v1.6.1).


Thank you @vladikoff
